modules/virt-creating-a-primary-cluster-udn.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-cluster-udn_{context}"]                                  
+= Creating a primary cluster-scoped user-defined network
+
+You can connect multiple namespaces to the same primary user-defined network (UDN) to achieve native tenant isolation.
+
+.Prerequisites
+. You have access to the cluster as a user with `cluster-admin` privileges.
+. You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Create a `ClusterUserDefinedNetwork` object to specify the custom network configuration:
++
+.Example `ClusterUserDefinedNetwork` manifest
+[source,yaml]
+----
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: cudn-l2-net # <1>
+spec:
+  namespaceSelector: # <2>
+    matchExpressions:
+    - key: kubernetes.io/metadata.name
+      operator: In
+      values:
+        - "mynamespace"
+        - "theirnamespace"
+  template:
+    topology: Layer2 # <3>
+    role: Primary # <4>
+    mtu: 9000
```

Comment:
This would not work on most of setups. Please remove the explicit MTU configuration to stick to the default. With that, the config will be easier to copy paste.